includes/api/class-edd-api.php

```diff
@@ -1157,6 +1157,7 @@ public function get_product_data( $product_info ) {
 		$product['info']['modified_date'] = $product_info->post_modified;
 		$product['info']['status']        = $product_info->post_status;
 		$product['info']['link']          = html_entity_decode( $product_info->guid );
+		$product['info']['permalink']     = get_permalink( $product_info->ID );
```

Comment:
Should the permalink be run through `html_entity_decode` the same way we are doing for the `guid` link?